packages/containers/src/List/List.connect.js

```diff
@@ -2,20 +2,18 @@ import get from 'lodash/get';
 import { List } from 'immutable';
 import { cmfConnect } from '@talend/react-cmf';
 import Container, { DEFAULT_STATE } from './List.container';
-import {
-	configureGetFilteredItems,
-	configureGetPagination,
-	configureGetPagedItems,
-} from './selector';
+import { configureGetPagination, configureGetPagedItems } from './selector';
 
 function componentId(ownProps) {
 	return ownProps.collectionId;
 }
 
-function getItems(state, config) {
-	const items = configureGetFilteredItems(config)(state);
-
-	return items || new List();
+function getItems(componentState, config) {
+	let items = config.items;
```

Comment:
It would be easier to reason about if the variable is not declared.
Just return the value conditionally.

Reply:
added condition to config.items to return empty List if items are not available initially

Reply:
What I was telling is about the variable reassignment which is not necessary, and come with more thinking about how it's manipulated et so what's really inside, where what really interest us is what the function return.

```javascript
if (componentState) {
    return componentState.get('items', config.items);
}
return config.items;
```

and if we know we have a state which would be a better idea :
```javascript
    return componentState.get('items', config.items);
```
which would have removed the need for a function at all ;)